pkg/codegen/go/gen_program.go

```diff
@@ -595,6 +595,9 @@ func (g *generator) lowerResourceOptions(opts *pcl.ResourceOptions) (*model.Bloc
 	if opts.IgnoreChanges != nil {
 		appendOption("IgnoreChanges", opts.IgnoreChanges, model.NewListType(model.StringType))
 	}
+	if opts.DeletedWith != nil {
+		appendOption("DeletedWith", opts.DeletedWith, model.StringType)
```

Comment:
Should this look more like `opts.Parent` since `DeletedWith` takes a `Resource` like `opts.Parent` does?

```suggestion
		appendOption("DeletedWith", opts.DeletedWith, model.DynamicType)
```

Reply:
It should 👍🏼